overcast-sonos.py

```diff
@@ -70,19 +73,22 @@ def getMetadata(id, index, count):
                 'id': 'episodes',
                 'title': 'All Active Episodes',
                 'itemType': 'container',
-                'canPlay': False
+                'canPlay': False,
+                'albumArtURI': 'http://is2.mzstatic.com/image/thumb/Purple62/v4/22/c7/93/22c793a7-55a8-e72b-756c-90641e7b96d4/source/175x175bb.jpg',
             }},
             {'mediaCollection': {
                 'id': 'podcasts',
                 'title': 'Podcasts',
                 'itemType': 'container',
-                'canPlay': False
+                'canPlay': False,
+                'albumArtURI': 'http://is2.mzstatic.com/image/thumb/Purple62/v4/22/c7/93/22c793a7-55a8-e72b-756c-90641e7b96d4/source/175x175bb.jpg',
             }},
         ]}
 
     elif id == 'episodes':
-        episodes = overcast.get_active_episodes()
-        response = {'getMetadataResult': [{'index': 0, 'count': len(episodes), 'total': len(episodes)}]}
+        all_episodes = overcast.get_active_episodes()
+        episodes = all_episodes[index:index+count]
+        response = {'getMetadataResult': [{'index': index, 'count': len(episodes), 'total': len(all_episodes)}]}
```

Comment:
Paging is sort of useless; Overcast limits it to 250 episodes usually and less than that in some cases. Sonos will only read 100 at a time, so this simply fixes that problem. In the future we could cache this somewhere to avoid the same network call multiple times, but it's not that important.